client/src/components-page/profile/Profile.js

```diff
@@ -33,7 +33,7 @@ const Profile = ({ currentProfile, loading, auth, match, getUserProfile }) => {
             <p className={style.friends}>Friends ({currentProfile.num_friends})</p>
             {
               auth.user.id === currentProfile.id 
-              ? <Button warning>Edit Profile</Button>
+              ? <Button warning onClick={() => history.push('/profile/edit')}>Edit Profile</Button>
```

Comment:
I think ternary operators are suited to 1 liners/short things. I'd use the normal `if/else` conditionals here

Reply:
Ehh I'm not a fan of if/else in JSX for something like this, looks too verbose imo. I could edit that down to 1 line but i formatted it that way more for general readability

Reply:
Can you tab some of this at least?
e.g.
https://github.com/airbnb/javascript#comparison--nested-ternaries